src/DynamoCoreWpf/ViewModels/Core/Converters/SerializationConverters.cs

```diff
@@ -78,6 +81,32 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
         }
     }
 
+    public class NodeViewModelWriteConverter : JsonConverter
+    {
+        public override bool CanConvert(Type objectType)
+        {
+            return objectType == typeof(NodeViewModel);
+        }
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            throw new NotImplementedException();
+        }
+
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        {
+            var directSerialization = JsonConvert.SerializeObject(value);
+            var jobject = JObject.Parse(directSerialization);
+            //if the nodeView's model is a symbol node add some extra data.
+            if ((value as NodeViewModel).NodeModel is Symbol)
+            {
+                jobject.Add("Summary", ((value as NodeViewModel).NodeModel as Symbol).Parameter.Summary);
```

Comment:
Can you check if empty summary will be kept and serialized as empty string? Just curious